db/migrate/20160203130657_add_rejection_message_to_trip.rb

```diff
@@ -1,5 +1,5 @@
 class AddRejectionMessageToTrip < ActiveRecord::Migration
   def change
-    add_column :trips, :rejection_message, :text
+  	add_column :trips, :rejection_message, :text, default: ''
```

Comment:
Naja, auf dem Branch gibt es nur noch die neue. Ich vermute mal, weil sie gleich heißen hat rails die alte durch die neue ersetzt. Und wenn es die alte noch geben würde, müsste in der neuen m. E. n. nach auch kein add_column stehen, denn die Spalte gibt es dann ja schon, sondern sowas wie change_column oder so.

Aber ich sehe gerade, die alte ist von gestern. Gab es diese nur auf deinem Branch oder war die schon auf dem dev oder so? Falls es diese Migration nur auf deinem Branch gibt dürfte nicht all zu viel passieren...